docs/DEV_ENVIRONMENT.md

```diff
@@ -38,6 +38,11 @@
        ```
 9. Try `cargo test --all --features run-docker-tests -- --test-threads=16`.
 
+   Note for MacOS users:
+    - Utxo test docker container may fail to start due to TCP port conflict with MacOS AirPlay Receiver (using the same 7000 port). Disabling AirPlay Receiver resolves this problem.
+
+    - Nucleus test docker container (and depending ibc-relayer container) may fail to start on MacOS. The MacOS Docker "Enable host networking" option may help with this issue.
```

Comment:
~~depending~~  dependent
<img width="645" height="154" alt="image" src="https://github.com/user-attachments/assets/4806f531-4c5d-4dd7-ae05-af9bbd1ca40a" />

- capitalisation of proper nouns (e.g. `AirPlay Receiver`) is valid, though this is inconsistently applied to `docker`/`Docker`. Given the context of its use, being referenced as an object (regular noun) rather than product, lowercase would be the way to go.

- Inconsistent capitalisation of acronyms: `Utxo`, `TCP`, `ibc`

Other wise good in terms of spelling/grammar etc. (a stickler would want more use of definite article "`the`", but its superfluous in a tech doc context).

Reply:
> but its superfluous in a tech doc context

~its~ it's

Reply:
Thanks for the notes, 
tried to fix: https://github.com/KomodoPlatform/komodo-defi-framework/pull/2544/commits/fe87b87ca59cba1a997f5d39593c35ceca6e322a